pages.zh/linux/apt.md

```diff
@@ -19,10 +20,14 @@
 
 `sudo apt install {{软件包}}`
 
-- 移除指定软件包（使用`purge`可以同时移除其配置文件）：
+- 移除指定软件包（使用 `purge` 同时移除其配置文件）：
 
 `sudo apt remove {{软件包}}`
 
 - 将所有已安装软件包更新到最新可用版本：
 
 `sudo apt upgrade`
+
+- 列出所有软件包：
+
+`apt list`
```

Comment:
the `apt list --installed` is in the main page but not in this translation.

Reply:
Done